app/components/post_draft/post_input/post_input.tsx

```diff
@@ -319,6 +321,7 @@ export default function PostInput({
             onChangeText={handleTextChange}
             onFocus={onFocus}
             onPaste={onPaste}
+            selection={canUpdateCursorPosition ? {start: cursorPosition, end: cursorPosition} : undefined}
```

Comment:
I don't completely get the idea here. Can you explain it?

I see that you set the `canUpdateCursorPosition` state based on who is updating the cursor position, but it would feel that the moment one of the systems sets this to true, we stop being able to select some part of the input.

Reply:
So, the need of updating cursor is when the user uses autocomplete or pick emoji. Whenever user, does the same, `canUpdateCursorPosition` sets to true but then after when user selects or type something, it changes to false and we no longer control selection. This way the selection doesn't gets over written

Reply:
@yasserfaraazkhan Please give some love to this part of the functionality when you get to QA this.